app/components/organen/organen-table.hbs

```diff
@@ -0,0 +1,39 @@
+<AuDataTable
+  @content={{@content}}
+  @isLoading={{@isLoading}}
+  @noDataMessage="Geen bestuursorganen gevonden"
+  @sort={{@sort}}
+  @page={{@page}}
+  @size={{@size}}
+  as |t|
+>
+  <t.content as |c|>
+    <c.header>
+      <AuDataTableThSortable
+        @field="naam"
+        @currentSorting={{@sort}}
+        @label="Naam"
+      />
+      <th>Huidige voorzitter</th>
+      <th>Aantal leden</th>
+      <th>Oprichtingsdatum</th>
+      <th>{{! Bewerk Action }}</th>
+      <th>{{! Archive Action }}</th>
+    </c.header>
+
+    <c.body as |row|>
+      <td>{{row.naam}}</td>
+      <td>TODO add voorzitter</td>
+      <td>TODO add aantal leden</td>
+      <td>
+        {{moment-format row.bindingStart "DD-MM-YYYY"}}</td>
```

Comment:
oprichtingsdatum isn't the same as the bindingStart. Binding Start is the start of the current legislature (bestuursorgaan in de tijd). I would remove this field for now as we don't have it in the model either

Reply:
Would it be a good idea to check the bestuursorganen in de tijd for the earliest start? or not that important now?